### DISCUSSION TITLE
How to change inventory field type

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16726] [How to] [answered]
I have necessity to convert inventory comment field to be text only. Where I can do it?

**Comment by Jianbin719** [comment]
How about changing UItype from database dirctly??

**Reply by s2b-git** [reply]
vtiger_field doesn't hold inventory fields. UIType is defined in code.
The wysiwig type of this field seems to be unnecessary (at least from my point of view). Disadvantage shows up when you adding lots of line items with comment on copy/paste basis. If you copy source text from some html, the format is kept and this messing tables in inventory PDF's.  Sure you can remove format with one click, but than why not pure text field? 
I mean unless someone want to  embed extra font and images, but again, table format is completely demolished.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
remove class `js-editor js-editor--basic`
https://github.com/YetiForceCompany/YetiForceCRM/blob/a1fa22a70a4c5d98df454e63634ed2330281c8a6/layouts/basic/modules/Vtiger/inventoryfields/EditViewComment.tpl#L8-L12

**Reply by s2b-git** [reply]
This is The Trick. Thank you very much!
